addon/components/sl-menu.js

```diff
@@ -396,6 +403,22 @@ export default Ember.Component.extend({
     }.on( 'init' ),
 
     /**
+     * Initialize initClassNames array
+     *
+     * @function initClassNames
+     * @observes "init" event
+     * @returns  {undefined}
+     */
+    initClassNames: function() {
+        var classNames = this.get( 'classNames' ),
+            extraClassNames = this.get( 'extraClassNames' );
+
+        if ( extraClassNames && extraClassNames.length > 0 ) {
```

Comment:
The check for extraClassNames should not be necessary since it has already been initialized as an array.  Regardless, this entire line should be replaced with an Ember.isNone/isEmpty or other call as appropriate.